packages/api/src/router/appSessions.ts

```diff
@@ -176,11 +135,11 @@ export const appSessionsRouter = router({
 
       await checkMutationPermissions(userId);
 
-      const id = nanoid(16);
+      const chatId = generateChatId();
 
       const output: AilaPersistedChat = {
-        id,
-        path: `/aila/${id}`,
+        id: chatId,
+        path: `/aila/${chatId}`,
```

Comment:
I don't think we should be storing path against the record, but will remove that another time